.github/workflows/copybara.yaml

```diff
@@ -18,23 +18,28 @@ on:
   push:
     branches:
     - copybara
+
 jobs:
   create_pr:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v3
-    - name: Checkout a pull request from copybara to main
+    - name: Check out repo
+      uses: actions/checkout@v3
+      with:
+        ref: 'copybara'
+    - name: Create pull request for changes
+      uses: googleapis/code-suggester@v2
       env:
-        GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
-      run: |
-        # Only create a PR if DNE
-        if ! [[ `gh pr list --base main --search "[Copybara]"` ]]; then
-          git config --global user.email "yoshi-code-bot@google.com"
-          git config --global user.name "Yoshi Code Bot"
-          gh pr create \
-          --base main \
-          --head copybara \
-          --title "[Copybara] Update with new internal changes from Piper" \
-          --body "This is a auto-generated pull request. The PR updates the default branch with new changes from the copybara branch. Refer to PR commit history for more details." \
-          --label "cla: yes"
-        fi
\ No newline at end of file
+        ACCESS_TOKEN: ${{ secrets.YOSHI_CODE_BOT_TOKEN }} # manually created and added as repo secret
```

Comment:
I'm wondering if this will help with the CLA issue? Currently when the PR is generated through  `GITHUB_TOKEN`, it always have the CLA issue (e.g. https://github.com/googleapis/google-cloudevents/pull/359) and we ended up manually creating a PR to bypass this check.

Reply:
This token would be associate with a bot email (used in other DPE repos) instead of GitHub action bot token that can't sign the CLA. My write up on this issue can be found [go/github-action-bot-cla](http://go/github-action-bot-cla)

Reply:
Make sense, thanks Averi!